simple/tests/stats/runner_test.py

```diff
@@ -125,6 +133,12 @@ def test_config_with_wildcards(self):
   def test_input_dir_driven(self):
     _test_runner(self, "input_dir_driven", is_config_driven=False)
 
+  def test_input_dir_driven_with_existing_old_schema_data(self):
+    _test_runner(self,
+                 "input_dir_driven_with_existing_old_schema_data",
+                 is_config_driven=False,
+                 input_db_file_name="sqlite_old_schema_populated.db")
```

Comment:
With binary `*.db` files, the tests can feel opaque. Is it possible to have the tests take csv files as input and compare against expected csv files?

Reply:
Sure, I can do that. Before I do so, are you able to review the non-test portions of the PR so I can make test updates all in one if necessary? Also so I can work during the time zone offset.

Reply:
Yes. The overall implementation looks good, added a couple of nits.

Reply:
Oh - this totally works. It's text based and a truer representation of a DB vs CSVs. 

Didn't know about the `iterdump()` utility. Seems like the exact thing we need here.